libraries/iterate/src/watchdog.js

```diff
@@ -22,7 +22,7 @@ class WatchDog extends events.EventEmitter {
       } else {
         // This is being safe rather than sorry
         debug('exiting becase there is no expired event listener');
-        process.exit(1); // eslint-disable-line no-process-exit
+        //process.exit(1); // eslint-disable-line no-process-exit
```

Comment:
Should we just remove this line and the debug before it?

Reply:
That happened in a later commit.  I didn't mean to commit the outcommenting, but it happened anyway!  I intend to squash this on landing -- the split commits were just a way for me to double back when I accidentally broke tests.